### ISSUE TITLE
Having trouble summiting my "Make hello"

### ISSUE THREAD

**Issue by Jackbe42** [078986/submit50-cs50-problems-2020-x-hello#1]
I do not know how to submit by make hello.

**Comment by cpena1999** [comment]
How did you submit your "Make hello" homework?

**Comment by molifin** [comment]
> I do not know how to submit by make hello.

**Comment by ameenmuhammad-693** [comment]
DEAR FIRST SIGN IN GITHUB ACCOUNT AND THEN OPEN THE CS50 SANDBOX AND AFTER
COMPLETING THE CODE YOU CAN COPY PASTE CHECKCS50 AND PUT USER NAME AND
PASSWORD OF GITHUB ACCOUNT AND THEN COPY AND PASTE SUBMIT CS50 YOUR TASK
HAVE BEEN SUBMITTED SUCCESSFULLY

**Comment by sraavyi** [comment]
I cannot figure how to submit at all!!

**Comment by sraavyi** [comment]
ive been struggling for a while trying submit my projects

**Comment by sraavyi** [comment]
I thought we were supposed to use ide not sandbox according to the 2020 summer class

**Comment by priya1877** [comment]
yeah can somebody pls explain because we were supposed to use IDE not Sandbox

**Comment by ameenmuhammad-693** [comment]
https://www.youtube.com/watch?v=cEINS4-X82A
here is the link for how to submit ide projects I suggest don't make too
much folder . If you make many folders here is the link for navigation
between foldershttps://www.youtube.com/watch?v=cJrma4ICmng
she defines preety much good
additional link https://www.youtube.com/watch?v=tEjbe85aZT4

<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by Hau519** [comment]
In IDE, in the terminal of your code. You can submit by type submit50 cs50/problems/2020/x/hello and then it will appear to log in you GITHUB account then you can submit.

**Comment by sraavyi** [comment]
I figured it out a while back. thank you though! 

Sraavya I
Wayzata 2021
